lib/widgets/yust_text_field.dart

```diff
@@ -123,7 +123,6 @@ class _YustTextFieldState extends State<YustTextField> {
   @override
   Widget build(BuildContext context) {
     if (widget.value != null &&
```

Comment:
If we were to also remove `widget.value != null`, a `null`-Value passed to the `value`-Property of the Widget would correctly update the Textfield to be empty.

@janneskoehler, was there a reason why we disallow passed null values to override the current content?